app/models/installed_technology.rb

```diff
@@ -276,8 +276,10 @@ def valid_profile?
     end
   end
 
-  def is_battery?
-    %w(congestion_battery households_flexibility_p2p_electricity).include?(type)
+  def no_profile?
+    %w(congestion_battery
```

Comment:
If this must be hard-coded, define it in a constant somewhere so that a new array and three strings aren't created every time `no_profile?` is called. However, this seems like it would be best as a boolean column in the `technologies` table (something like `requires_profile default=true null=false ...`).

My general rule is that if I ever find yourself hard-coding the name of something from the database, especially unique keys such as these, try to find a way to move the information into the database and out of the code. Otherwise the application becomes very fragile to changes; if ever the names of these technologies were to change, `no_profile?` would return the wrong value.